auction-app-backend/src/main/java/com/example/auctionapp/controller/AuthController.java

```diff
@@ -37,18 +39,18 @@ public AuthController(final AuthService authService) {
     }
 
     @PostMapping("/register")
-    public User register(@RequestBody final UserRequest user) {
+    public User register(@Valid @RequestBody final UserRequest user) {
         return authService.signUp(user);
     }
 
     @PostMapping("/login")
-    public JwtResponse login(@RequestBody final LoginRequest loginRequest, HttpServletResponse response) {
+    public String login(@Valid @RequestBody final LoginRequest loginRequest, HttpServletResponse response) {
```

Comment:
This should still be an object, let's avoid returning default data types as response. We should have our own.